src/data/locales/en.ts

```diff
@@ -2,7 +2,7 @@ export default {
   "profile.trustButton": "Trust",
   "profile.trustedMember": "Trusted",
   "profile.trustFailure": "Trusting failed, try again",
-
+  "profile.memberNotFound": "Sorry, member with username {memberUsername} not found",
```

Comment:
I don't particularly like having an apologetic tone, I'd rather just cut out the "Sorry". But since we don't have any strategy with how we're wording stuff doesn't really matter.